sci-physics/fastjet-contrib/fastjet-contrib-1.049.ebuild

```diff
@@ -23,7 +25,8 @@ PATCHES=(
 )
 
 src_configure() {
-	./configure --prefix=/usr --fastjet-config=/usr/bin/fastjet-config CFLAGS="${CFLAGS}" CXXFLAGS="${CXXFLAGS}" FFLAGS="${FFLAGS}" LDFLAGS="${LDFLAGS}" || die
+	CXX=`tc-getCXX`
```

Comment:
We prefer $() syntax but you shouldn't need this actually. Instead:
tc-export CXX